lib/screens/home/pages/members/bloc/members_page_bloc.dart

```diff
@@ -25,47 +26,66 @@ class MembersPageBloc extends Bloc<MembersPageEvent, MembersPageState> {
     }
   }
 
-  Stream<MembersPageState> _mapEventToMembersShowed(MembersPageEvent event) async* {
+  Stream<MembersPageState> _mapEventToMembersShowed(
+      MembersPageEvent event) async* {
     final currentState = state;
 
     if (event is MembersPageShowed && !_hasReachedMax(currentState)) {
       try {
         if (currentState is MembersPageInitial) {
           yield MembersPageLoading();
-          final List<User> users = await userRepository.getVerifiedUsers(pageNumber);
-          yield MembersPageSuccess(users: users, hasReachedMax: false);
+          final List<User> users =
+              await userRepository.getVerifiedUsers(pageNumber);
+          final User currentUser = await userRepository.getCurrentUser();
+          yield MembersPageSuccess(
+            users: users,
+            hasReachedMax: false,
+            currentUser: currentUser,
+          );
         }
         if (currentState is MembersPageSuccess) {
-          final users =
-              await userRepository.getVerifiedUsers((currentState.users.length ~/ 10) + 1);
+          final users = await userRepository
+              .getVerifiedUsers((currentState.users.length ~/ 10) + 1);
+          final User currentUser = await userRepository.getCurrentUser();
```

Comment:
@yashhalgaonkar I'm talking about this line?

Reply:
this is the event added when the user scrolls to the end of the page to load more users

Reply:
Okay. I am looking into this.

Reply:
Should I use sqflite to store the current user info in local storage? This will make it easy to get current user info in all other parts of app as well, solving the problem once and for all.

Reply:
sure, give it a try